frontend/.env.test

```diff
@@ -0,0 +1,4 @@
+DOMAIN=http://localhost:8080
+REACT_APP_ENCODER=$DOMAIN/encoder
+REACT_APP_DECODER=$DOMAIN/decoder
+REACT_APP_STREAM=$DOMAIN/stream
```

Comment:
Does this grab from system environment variables if they are defined? If so, we should prefix these with `SWITCHBOARD_APP` to avoid conflicts. If not, forget about it.
Ex. `SWITCHBOARD_APP_DOMAIN`, `SWITCHBOARD_APP_ENCODER`, etc.

Reply:
Custom environment variables must start with `REACT_APP_`, so, it rules out the possibility of the proposed suggestion.